src/consts.ts

```diff
@@ -11,7 +11,11 @@ export const UriPrefix = '/';
 export const TitleDefault = 'Aleph Zero Staking Dashboard';
 export const DappName = 'Aleph Zero Staking Dashboard';
 export const PolkadotUrl = 'https://alephzero.org/staking/';
-export const DefaultNetwork = 'alephzero';
+export const DefaultNetwork =
+  process.env.NODE_ENV === 'production' &&
+  process.env.REACT_APP_DISABLE_MAINNET !== '1'
+    ? 'alephzero'
```

Comment:
It will be kind of weird if we set `REACT_APP_DISABLE_MAINNET == 1` but run in `dev`-mode b/c we will still default to Mainnet.

Reply:
We will default to mainnet only if it's a production build and the flag is not present. So in dev mode it'll always default to testnet. Or am I missing something?

Reply:
You're using _and_ (`&&`) so if any of the conditions is `false` we will default to `alephzerotestnet`, right?